sdk/playwrighttesting/microsoft-playwright-testing/review/microsoft-playwright-testing.api.md

```diff
@@ -38,6 +38,7 @@ export const getServiceConfig: (config: PlaywrightConfigInput, options?: Playwri
 // @public
 export interface MPTReporterConfig {
     enableGitHubSummary?: boolean;
+    enableMPTReporting?: boolean;
```

Comment:
This field has been added. However, it's not a breaking change since we haven't published the existing package yet.

Reply:
No worries, adding a new field is rarely a breaking change anyway and breaking changes between betas are acceptable - it's more a concern when making a breaking change between GA versions 👍 

I did have a question re: MPT acronym. I get that MicrosoftPlaywrightTesting is way too long but could this just be `enableReporting`? Or even more descriptive like `enablePortalUploads`, `uploadToPortal` or something? 

`enableReporting` seems not that descriptive, and `enableMPTReporting` doesn't really add any clarity on top of that. Feel free to ignore, just wanted to see if we can think of clearer names here...

Reply:
switched to enableResultPublish